locale/en/corteza-webapp-compose/block.yaml

```diff
@@ -419,7 +419,8 @@ recordRevisions:
   viewer:
     errors:
       no-record: No record found to display revisions for
-      disabled-on-module: Record revisions are disabled on this module
+      disabled-on-module: Record revisions are disabled on this module.
+      enabled-on-module: Record revisions are enabled on this module. To see the record revisions for a specific record, go to the public page for that record.
```

Comment:
No need for such long text, just say why the block cannot be displayed. Which is because the is no record